.circleci/config.yml

```diff
@@ -4,7 +4,10 @@ jobs:
   lint_and_test:
     working_directory: ~/cob_datapipeline
     docker:
-      - image: cimg/python:3.7.11
+      - image: cimg/python:3.9.10
+        auth:
+          username: $DOCKERHUB_USER
+          password: $DOCKERHUB_PASSWORD
```

Comment:
Circle was complaining about username and password not being set so that's why that's here.